image_to_text.py

```diff
@@ -35,7 +35,7 @@ class ImageToText(object):
     # gray scale level values from:
     # http://paulbourke.net/dataformats/asciiart/
     # 70 levels of gray
-    gscale1 = "$@B%8&WM#*oahkbdpqwmZO0QLCJUYXzcvunxrjft/\|()1{}[]?-_+~<>i!lI;:,\"^`'. "
+    gscale1 = "$@B%8&WM#*oahkbdpqwmZO0QLCJUYXzcvunxrjft/\|()1{}[]?-_+~<>i!lI;:,\"^`'. "  # noqa: W605
```

Comment:
It's better to make this an `r` string rather than add the `noqa`: `r"$@B%8&WM#*oahkbdpqwmZO0QLCJUYXzcvunxrjft/\|()1{}[]?-_+~<>i!lI;:,\"^`'."` Maybe also a `"""` string too so we don't need to escape the `"`